### PR TITLE
fix relative path error for skipping secrets checks and check which check id should be skipped

### LINKED ISSUE #1363: check CKV_SECRET_6 fails even when suppressed
**Describe the bug**
Checkov raises FAILED check CKV_SECRET_6 when running against a project even if the check is suppressed. However if running checkov on a single terraform file it doesn't flag the same error.

**To Reproduce**
Steps to reproduce the behaviour:
1. Create a single terraform file and save in under a new dir (test/main.tf) with a resource would fail this check but make a comment so checkov suppresses the alert:
```
resource "aws_emr_cluster" "cluster" {
  #checkov:skip=CKV_SECRET_6:todo
  #checkov:skip=CKV2_AWS_7:todo
  name          = "emr"
  termination_protection = true
  ebs_root_volume_size = 10
  service_role = "emr_role"
  master_instance_fleet {
    name = "emr-leader"
    target_on_demand_capacity = 1
    instance_type_configs {
      instance_type = var.emr_leader_instance_type

      ebs_config {
        size = "64"
        type = "gp2"
        volumes_per_instance = 1
      }
      configurations {
        classification = "spark-defaults"
        properties = {
          "spark.cassandra.connection.host" = "cassandra.domain.co.uk",
          "spark.cassandra.connection.port" = "9999",
          "spark.cassandra.auth.username" = "user123",
          "spark.cassandra.auth.password" = "pass123", #checkov:skip=CKV_SECRET_6:todo
        }
      }
    }
  }
}
```
2. Run cli command  `checkov -f test/main.tf`
3. Observe result behaves as expected:
```
...
 secrets scan results:

Passed checks: 0, Failed checks: 0, Skipped checks: 1

Check: CKV_SECRET_6: "Base64 High Entropy String"
	SKIPPED for resource: d34df506cfa43c623b4da8853ce91637ed575be6
	Suppress comment: checkov:skip=
	File: /test/main.tf:29-30
	Guide: https://docs.bridgecrew.io/docs/git_secrets_6
```
4. Run cli command `checkov --directory test/`
5. Observe the same test fail:
```
...
secrets scan results:

Passed checks: 0, Failed checks: 1, Skipped checks: 0

Check: CKV_SECRET_6: "Base64 High Entropy String"
	FAILED for resource: d34df506cfa43c623b4da8853ce91637ed575be6
	File: /main.tf:29-30
	Guide: https://docs.bridgecrew.io/docs/git_secrets_6

		29 |
```

**Expected behaviour**
There is no terraform code difference, checkov should behave the same way in both tests, it should show SKIPPED for CKV_SECRET_6.

**Desktop (please complete the following information):**
 - Terraform 1.0.1
 - Checkov Version 2.0.259

**Additional context**
I have not been able to observe if this behaviour is unique to check CKV_SECRET_6 or not. But it's the first time I have observed a suppressed check still show as failure.

### PR DESCRIPTION
**Pull request by gruebel**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

Fix #1363. the `path.join()` with `root_folder` and `secret.filename` is redundant, because the `secret.filename` has already the joined path stored. I also fixed an issue, when someone adds a skip comment, but the wrong `check_id`.

### CODE REVIEW

**Review by nimrodkor — APPROVED** [review]
@gruebel Tnx!

**Comment by nimrodkor** [comment]
Was it fixed in #1371 ?
If not - please fix the conflicts 🙏

**Comment by gruebel** [comment]
@nimrodkor I think the main problem was fixed by @kartikp10 this actually fixes an other issue regarding adding a skip comment, but with a wrong `check_id`, that's why I added the extra skip test and I also cleaned up the code a bit 😄

File: tests/secrets/test_runner.py [hunk 1]
```diff
@@ -63,7 +80,7 @@ def test_runner_multiple_files(self):
         self.assertEqual(4, len(report.failed_checks))
```
Comment (nimrodkor) [inline tests/secrets/test_runner.py:80]: @gruebel After merging from master, can you update this to 5 please?

Comment (gruebel) [reply tests/secrets/test_runner.py:80]: yeah sure, give me a few minutes 😄

**Pull request merged**